Dockerfile

```diff
@@ -1,9 +1,26 @@
-FROM python:3.10-alpine
+FROM python:3.9-slim as base
```

Comment:
I'm curious why you switched to 3.9 instead of 3.10

Reply:
the loop argument is deprecated at 3.10, so we are not yet ready 3.10.

Reply:
I think we need to merge this file ASAP to change our docker version work fine.
Maybe create another PR about this and merge ASAP ?

Reply:
Will this be fix in https://github.com/bluet/proxybroker2/pull/88 ?